dev/doc/critical-bugs

```diff
@@ -347,6 +347,16 @@ Conflicts with axioms in library
   GH issue number: none
   risk:
 
+  component: primitive floating-points
+  summary: Incorrect specification of PrimFloat.leb
+  introduced: 8.11
+  impacted released versions: 8.11.0, 8.11.1
```

Comment:
Also 8.11.2.

Reply:
Indeed

Reply:
You didn't push any change @proux01, despite having marked this as resolved.

Reply:
Oups, thanks, here it is.